specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/Certificate_Get.json

```diff
@@ -10,7 +10,7 @@
     "200": {
       "headers": {},
       "body": {
-        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/testcontainerenv/certificate-firendly-name",
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/testcontainerenv/certificates/certificate-firendly-name",
         "name": "certificate-firendly-name",
```

Comment:
I notice that in all of the places in this PR we have "firendly" instead of "friendly"; looks like it existed before this PR but we should maybe fix the spelling error. 😄